canadian-journal-of-soil-sciences.csl

```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="en-US" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Canadian Journal of Soil Science</title>
+    <id>http://www.zotero.org/styles/canadian-journal-of-soil-sciences</id>
```

Comment:
This should be `<id>http://www.zotero.org/styles/canadian-journal-of-soil-sciences</id>` ("science", not "sciences"). Idem for file name and "self" link.

http://www.nrcresearchpress.com/journal/cjss